packages/katello/katello/katello-backup-rotate-tar.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+## For this script it's advisable to use a shell, such as Bash,
+## that supports a TAR_FD value greater than 9.
+
+# This script taken from the tar info doc
+
+name=`expr ${TAR_ARCHIVE} : '\(.*\)\..*'`
+volnum=`printf "%03d" ${TAR_VOLUME}`
```

Comment:
The zero padding works, good point on the sorting. I wonder if we should go to 4 places like you originally had it? Users have databases in the TBs, so breaking into 1000+ files is not inconceivable